tests/python/gaia-ui-tests/gaiatest/apps/marketplace/app.py

```diff
@@ -70,3 +72,57 @@ def tap_install_button(self):
 
     def get_app_name(self):
         return self.root_element.find_element(*self._name_locator).text
+
+class MarketplaceDev(Marketplace):
+
+    name = 'Dev'
+    manifest_url = 'https://marketplace.firefox.com/app/bee10fbb-7829-4d99-8a9f-a6d0a905138e/manifest.webapp'
+
+    _enable_addons_locator = (By.ID, 'enable-addons')
+
+    def _enable_addons(self):
+        element = Wait(self.marionette).until(
```

Comment:
I'm only using `element` locally. I find it generally easier to work then with a generic name than to use special names for variables which I only use locally and don't reuse at all.